### ISSUE TITLE
2.0.81 version error - $ az webapp list-runtimes

### ISSUE THREAD

**Issue by CodingSta** [Azure/azure-cli#12228]
## Describe the bug

**Command Name**
`az webapp list-runtimes`

**Errors:**
```
'bool' object has no attribute 'provider'
Traceback (most recent call last):
python3.8/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
cli/command_modules/appservice/custom.py, ln 2159, in stacks
    self._load_stacks()
cli/command_modules/appservice/custom.py, ln 2180, in _load_stacks
    raw_stacks = self._client.provider.get_available_stacks(os_type_selected=os_type, raw=True)
AttributeError: 'bool' object has no attribute 'provider'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az webapp list-runtimes`

## Environment Summary
```
macOS-10.15.3-x86_64-i386-64bit
Python 3.8.1
Shell: bash

azure-cli 2.0.81
azure-cli-ml 1.0.83
azure-cli-iot-ext 0.7.1

Extensions:
azure-cli-ml 1.0.83
azure-cli-iot-ext 0.7.1
interactive 0.4.3

```

**Comment by arrownj** [comment]
@CodingSta , thanks for reporting, we are looking at it.

**Comment by panchagnula** [comment]
Same as https://github.com/Azure/azure-cli/issues/12159 please use Edge builds the fix will be available with 2.0.82 release.
